client/src/main/java/org/spine3/protobuf/TypeToClassMap.java

```diff
@@ -74,7 +74,12 @@
      *
      * <p>For example, for a key {@code spine.base.EventId}, there will be the value {@code org.spine3.base.EventId}.
      */
-    private static final BiMap<TypeName, ClassName> NAMES_MAP = Builder.build();
+    private static final BiMap<TypeName, ClassName> TYPE_TO_CLASS_MAP = Builder.build();
+
+    /**
+     * The inverse view of the primary type to class map.
+     */
+    private static final BiMap<ClassName, TypeName> CLASS_TO_TYPE_MAP = TYPE_TO_CLASS_MAP.inverse();
```

Comment:
We don't need to keep the reference of the inverse map. The `BiMap` assumes it. Have a look at `AbstractBiMap` implementation:

```java
   public BiMap<V, K> inverse() {
        return this.inverse;
    }
```